packages/psss-server/src/routes/reports/ConfirmCountryWeeklyReportRoute.ts

```diff
@@ -37,12 +56,93 @@ export class ConfirmCountryWeeklyReportRoute extends Route {
     const answers = mapUnconfirmedReportToConfirmedAnswers(report.results[0]);
 
     return this.meditrakConnection?.updateOrCreateSurveyResponse(
-      SURVEY_CODE,
+      CONFIRMED_WEEKLY_SURVEY_COUNTRY,
       countryCode,
       week,
       answers,
     );
   }
+
+  async createAlerts(countryCode: string, week: string) {
+    const report = await this.reportConnection?.fetchReport(
+      CONFIRMED_WEEKLY_REPORT_CODE,
+      [countryCode],
+      [week],
+    );
+
+    if (!report || report.results.length === 0) {
+      throw new RespondingError(
+        `Cannot create alerts: no confirmed weekly data found for ${countryCode} - ${week}`,
+        500,
+      );
+    }
+
+    const [result] = report.results;
+    const response: any = {
+      createdAlerts: [],
+    };
+    const startWeek = dateStringToPeriod(MIN_DATE, 'WEEK');
+    const activeAlertsData = await this.reportConnection?.fetchReport(
```

Comment:
I suppose we don't definitely have to decide how to deal with this case for now, as Outbreaks are out of scope, however are we sure that we should be creating a new alert if there is an existing (non-archived) Outbreak with that syndrome? I'd expect we don't (so this report would also need to return Outbreaks) but we can confirm with Erin on this.

Regardless not gonna suggest we bother creating a new report for this as of yet since Outbreaks are out of scope...

Reply:
I've actually confirmed this with Erin and I think she said we still should create a new alert if there's already an existing active outbreak.

This is from her:
```
Thus the only way a new one is created when there is already an alert for that syndrome in recent weeks is if the existing alert was archived or turned into an outbreak
```

Reply:
Tbh I'm still not 100% sure that logic is valid, but I'll discuss this with Erin when we want to implement outbreaks 👍